t/blocker-Databases.t

```diff
@@ -39,7 +39,7 @@ my $mock_elevate = Test::MockFile->file('/var/cpanel/elevate');
         $db->_blocker_mysql_upgrade_in_progress(),
         {
             id  => q[Elevate::Blockers::Databases::_blocker_mysql_upgrade_in_progress],
-            msg => "MySQL upgrade in progress. Please wait for the MySQL upgrade to finish.",
+            msg => "MySQL/MariaDB upgrade in progress. Please wait for the upgrade to finish.",
```

Comment:
s/MySQL/MariaDB upgrade/Database server/?

Reply:
Considering it. I think I was worried about getting customers confused between an upgrade of MySQL or MariaDB initiated through WHM and the PostgreSQL upgrade I just finished implementing.

Reply:
Never mind: [it has the same issue as the Profiles](https://docs.cpanel.net/whm/sql-services/mysql-or-mariadb-upgrade/). So it's staying.